detox/src/logger/__snapshots__/DetoxLogger.test.js.snap

```diff
@@ -96,6 +96,7 @@ exports[`DetoxLogger - main functionality - should log complete duration events
 00:00:00.000 detox[PID] E  Debug duration
 00:00:00.000 detox[PID] B  Trace duration
 00:00:00.000 detox[PID] E  Trace duration
+00:00:00.000 detox[PID] B  Fatal (pending)
```

Comment:
If it does not treat it as a promise, it would print erroneously the "E" (end) event for it.